src/store.c

```diff
@@ -589,6 +590,26 @@ static int p11prov_store_set_ctx_params(void *pctx, const OSSL_PARAM params[])
     return RET_OSSL_OK;
 }
 
+void p11prov_store_quickload(void *provctx, const char *uri,
```

Comment:
There isn't really anything quick about this call :-)
This is borderline bike-shedding but I would like something more clear like:
p11prov_store_direct_fetch() or maybe p11prov_store_internal() ...